app/views/thesis/new.html.erb

```diff
@@ -102,7 +103,22 @@
       $("#files_to_upload").html(fileblurb + '<ul>' + filenames + '</ul>');
   });
   </script>
-  <script>$("#new_thesis").validate();</script>
+  <script>
+    $("#new_thesis").validate({
+      invalidHandler: function(event, validator) {
+        var errors = validator.numberOfInvalids();
+        if (errors) {
+          var message = errors == 1
+            ? 'You missed 1 field. It has been highlighted'
```

Comment:
The form was not submitted successfully - one required field needs to be fixed. 
The form was not submitted successfully - X required fields need to be fixed.

Reply:
will change then merge. Thx!